src/System.Runtime.Extensions/tests/System/Math.cs

```diff
@@ -755,6 +755,9 @@ public static void Ceiling_Double(double value, double expectedResult, double al
         [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
         public static void Ceiling_Double_IEEE(double value, double expectedResult, double allowedVariance)
         {
+            if (expectedResult == -0.0D && PlatformDetection.IsNetNative)
```

Comment:
Should we just skip the whole test (or move the relevant info data's to another test method that just calls this one and that's skipped appropriately)?

Reply:
It would also be good to fix this on .NET Native. The issue here was with an MSVC optimization, so we should just need to disable for uap as well.